spec/inputs/file_spec.rb

```diff
@@ -8,11 +8,11 @@
 
   delimiter = (LogStash::Environment.windows? ? "\r\n" : "\n")
 
-  describe "starts at the end of an existing file" do
+  it "should starts at the end of an existing file" do
     tmpfile_path = Stud::Temporary.pathname
     sincedb_path = Stud::Temporary.pathname
 
-    config <<-CONFIG
+    conf = <<-CONFIG
```

Comment:
Might be just a naming question in here, so please feel free to ignore me. Why the rename to conf?